int8 Ch
---
int16 Val_VOLT